File: 17.c
#include <stdio.h>

int main() 
{
    double x, y;
    char a;
    printf("Enter first number: ");
    scanf("%lf", &x);
    printf("Enter second number: ");
    scanf("%lf", &y);
    printf("Enter operation (+, -, *, /): ");
    scanf(" %c", &a);
    if (a == '+') 
    {
        printf("Result: %lf\n", x + y);
    }
    else if (a == '-') 
    {
        printf("Result: %lf\n", x - y);
    }
    else if (a == '*') 
    {
        printf("Result: %lf\n", x * y);
    }
    else if (a == '/') 
    {
        if (y != 0)
            printf("Result: %lf\n", x / y);
        else
            printf("Can't be divided by zero\n");
    }
    return 0;
}

File: 17.js
const readline = require("readline");
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});
let inputs = [];
rl.question("Enter first number: ", (num1) => {
    inputs.push(parseFloat(num1));
    rl.question("Enter second number: ", (num2) => {
        inputs.push(parseFloat(num2));
        rl.question("Enter operation (+, -, *, /): ", (op) => {
            const [x, y] = inputs;
            let result;
            switch (op) {
                case '+':
                    result = x + y;
                    break;
                case '-':
                    result = x - y;
                    break;
                case '*':
                    result = x * y;
                    break;
                case '/':
                    result = y !== 0 ? x / y : "Can't be divided by zero";
                    break;
            }
            console.log("Result:", result);
            rl.close();
        });
    });
});
